pandas/tests/io/json/test_pandas.py

```diff
@@ -1275,11 +1274,8 @@ def test_to_json_large_numbers(self, bigNum):
         expected = '{"0":{"articleId":' + str(bigNum) + "}}"
         assert json == expected
 
-    @pytest.mark.parametrize("bigNum", [sys.maxsize + 1, -(sys.maxsize + 2)])
-    @pytest.mark.skipif(not compat.IS64, reason="GH-35279")
+    @pytest.mark.parametrize("bigNum", [-(2 ** 63) - 1, 2 ** 64])
     def test_read_json_large_numbers(self, bigNum):
-        # GH20599
```

Comment:
is this no longer relevant?

Reply:
I wonder if keeping references to issues is a good thing. I could find a couple that could be referenced here, including the one related to this PR. By deleting the comment, my aim is to only focus on what is needs testing. If you agree with this, I would keep it this way.

Reply:
these comments can be very helpful and its hard to imagine them outright being harmful

Reply:
Re-introduced the commented info.